src/Workspaces/Core/Portable/Indentation/FormattingOptions.IndentStyle.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.CodeAnalysis.Formatting
 {
     public static partial class FormattingOptions
     {
+        // Publicly exposed.  Keep in sync with IndentStyle2 in the CodeStyle layer.
```

Comment:
```suggestion
        // Publicly exposed.  Keep in sync with FormattingOptions2.IndentStyle in the CodeStyle layer.
```

Reply:
Handled in https://github.com/dotnet/roslyn/pull/60462